src/content/accessibilityTests/accessibility-configure.md

```diff
@@ -0,0 +1,229 @@
+
+```tsx title="RestaurantCard.stories.ts|js"
+// Replace your-framework with the framework you are using (e.g., nextjs, vue3-vite)
+import type { Meta, StoryObj } from "@storybook/your-framework";
+import { restaurants } from "../../stub/restaurants";
+import { RestaurantCard } from "./RestaurantCard";
+
+const meta = {
+  title: "Components/RestaurantCard",
+  component: RestaurantCard,
+  parameters: {
+    a11y: {
+      config: {
+        rules: [
+          {
+            id: "minimum-paragraph-length",
+            selector: "p",
+            impact: "critical",
+            all: ["paragraph-minimum-text"],
+            any: [],
+            none: [],
+            metadata: {
+              description: "Ensures paragraphs have meaningful content",
+              help: "Paragraphs should contain at least 5 characters",
+            },
+          },
+        ],
+        checks: [
+          {
+            id: "paragraph-minimum-text",
+            evaluate: function evaluate(node: HTMLParagraphElement) {
+              const textContent = node.textContent?.trim();
+              return textContent && textContent.length > 0
+                ? textContent.length > 50
+                : false;
+            },
+          },
+        ],
+      },
+    },
+  },
+} satisfies Meta<typeof RestaurantCard>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
+  args: {
+    ...restaurants[0],
+    name: "Burger Kingdom",
+  },
+} satisfies Story;
+```
+
+## Disable accessibility tests
+
+### Both in Storybook and Chromatic
+
+For Storybook 8+, use the [a11y global](https://storybook.js.org/docs/writing-tests/accessibility-testing#turn-off-automated-a11y-tests) to disable accessibility tests for a story and/or component, both in Storybook and Chromatic.
+
+```tsx title="MyComponent.stories.ts|tsx"
+// Adjust this import to match your framework (e.g., nextjs, vue3-vite)
+import type { Meta, StoryObj } from "@storybook/your-framework";
+import { MyComponent } from "./MyComponent";
+
+const meta: Meta<typeof MyComponent> = {
+  component: MyComponent,
+};
+
+export default meta;
+type Story = StoryObj<typeof MyComponent>;
+
+export const NonA11yStory: Story = {
+  globals: {
+    a11y: {
+      // This option disables all automatic a11y checks on this story
+      manual: true,
+    },
+  },
+} satisfies Story;
+```
+
+<div class="aside">
+
+For older versions of Storybook, use [a11y parameter](https://storybook.js.org/docs/7/writing-tests/accessibility-testing#how-to-disable-a11y-tests) instead.
+
+</div>
+
+### Only in Chromatic
+
+To disable Accessibility Tests only within Chromatic you can use the [`chromatic.disableSnapshot` parameter](https://www.chromatic.com/docs/disable-snapshots/#with-storybook). Note, this will also disable visual tests for this story and/or component.
```

Comment:
Shouldn't this be the following link to align with the rest of the documentation? I don't see any reason why we should have the full URL
```suggestion
To disable Accessibility Tests only within Chromatic you can use the [`chromatic.disableSnapshot` parameter](/docs/disable-snapshots/#with-storybook). Note, this will also disable visual tests for this story and/or component.
```